application/features/bootstrap/fakes/FakeEmailer.php

```diff
@@ -38,28 +38,32 @@ public function forgetFakeEmailsSent()
      *
      * @param string $messageType The type of message.
      * @param User $user The User in question.
+     * @param string $address Email address to find. If provided, overrides $user->email.
      * @return array[]
      */
     public function getFakeEmailsOfTypeSentToUser(
```

Comment:
Rather than adding a 3rd parameter (which, when present, makes the 2nd parameter unused), what would you think about changing this method to be this?
```php
public function getFakeEmailsOfTypeSentToAddress(string $messageType, string $emailAddress)
```